db/schema.rb

```diff
@@ -31,7 +31,7 @@
   add_index "events", ["user_id"], name: "index_events_on_user_id", using: :btree
 
   create_table "users", force: :cascade do |t|
```

Comment:
Oh cool. `force: :cascade` is new in Rails 4.2 I guess: https://github.com/rails/rails/pull/18082